extensions/wikia/AssetsManager/config.php

```diff
@@ -720,6 +720,15 @@
 	)
 );
 
+$config['mobile_krux_js'] = array(
+	'type' => AssetsManager::TYPE_JS,
+	'assets' => array(
+		'//resources/wikia/modules/scriptwriter.js',
+		'//extensions/wikia/AdEngine/js/Krux.js',
+		'//extensions/wikia/AdEngine/js/Krux.run.js',
```

Comment:
Same as with interstitial and bottom_leaderboards... should we find another solution for this?